pkg/api/v1/clients/kube/controller/controller.go

```diff
@@ -138,7 +138,7 @@ func (c *Controller) processNextWorkItem() bool {
 		}
 		switch e.eventType {
 		case added:
-			c.handler.OnAdd(e.new)
+			c.handler.OnAdd(e.new, false)
```

Comment:
Can you add a comment why `false` is provided (ie why not true?)

Reply:
done